carapace-server/src/main/java/org/carapaceproxy/server/certiticates/DynamicCertificatesManager.java

```diff
@@ -325,7 +325,10 @@ private void certificatesLifecycle() {
                     groupMembershipHandler.fireEvent(EVENT_CERT_AVAIL_CHANGED);
                 }
             } catch (AcmeException | IOException | ConfigurationStoreException | GeneralSecurityException | IllegalStateException ex) {
+                LOG.log(Level.SEVERE, null, ex);            
+            } catch (Throwable ex) {
                 LOG.log(Level.SEVERE, null, ex);
```

Comment:
catch  Throwable is a code smell
and re-throwing it is also bad

Reply:
yep but maybe is the only way to find out what happens... i'll remove it asap